### ISSUE TITLE
Dependency conflict

### ISSUE THREAD

**Issue by XayOn** [XayOn/torrentmirror#1]
Torrentmirror depending on a specific version of robobrowser is breaking packaging to arch of katcr. This should be addressed here.

**Comment by BachoSeven** [comment]
Summarizing what said in https://github.com/XayOn/katcr/issues/129:
- on Arch, robobrowser pulls in the latest `werkzeug`, which breaks `robobrowser` ([1](https://github.com/jmcarp/robobrowser/issues/96), [2](https://github.com/jmcarp/robobrowser/issues/93)).
- a possible solution would be to allow using the newer version of `werkzeug` through this small change:
``` py
import werkzeug
werkzeug.cached_property = werkzeug.utils.cached_property
```
The above could be added before importing `robobrowser`, as a workaround for that breaking change. It should be enough to do so [here](https://github.com/XayOn/torrentmirror/blob/9e09f994f34c51ad37e4662970a02c173b3c45a4/torrentmirror/__init__.py#L8).

**Comment by XayOn** [comment]
@BachoSeven torrentmirror 2.0.3 has been released with the fix, and updated werkzeug versions.

I've also made another minor fix due to a change in torrentmirror.net
I'm updating katcr's deps now.

**Comment by XayOn** [comment]
@BachoSeven I just published katcr 3.0.6, thanks! 👍🏽

**Comment by BachoSeven** [comment]
Thank you, I just published `katcr` as an AUR package because it built, but I believe there are some things not working as they should: selecting a torrent from the search view gives me the magnet link, but it is followed by this warning:
```
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f6a84f68be0>
```
or this similar but longer one:
```
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f579666ed00>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f579669c700>, 11646.288644454)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f579663f0a0>, 11646.586501255)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f579663f760>, 11646.856275767)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f579665b0a0>, 11647.60878818)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f579665b5e0>, 11647.907814565)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f579665b9a0>, 11648.170023446)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d38520>, 11648.616481016)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d38b80>, 11649.115423636)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d38040>, 11649.460662387)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d44b80>, 11649.779933047)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d38fa0>, 11650.047641389)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d444c0>, 11650.167494221)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d54640>, 11650.395983883)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d549a0>, 11650.72364622)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d610a0>, 11651.212468246)]', '[(<aiohttp.client_proto.ResponseHandler object at 0x7f5795d54580>, 11651.402663689)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f579666ef40>
```

Perhaps it's connected to this warning during build tests?
![image](https://user-images.githubusercontent.com/55460785/113522734-57317900-95a3-11eb-94aa-3cc7f01aed30.png)

**Comment by XayOn** [comment]
@BachoSeven Yep, https://github.com/XayOn/katcr/issues/144
Already fixed, it's only a warning but it's ugly to show it to users, I released katcr 3.0.7 with it.

**Comment by BachoSeven** [comment]
Perfect, thanks!
